hydra-node/test/Hydra/Network/ReliabilitySpec.hs

```diff
@@ -301,3 +312,27 @@ mockMessagePersistence numberOfParties = do
       , loadMessages = toList <$> readTVarIO messages
       , appendMessage = \msg -> atomically $ modifyTVar' messages (|> msg)
       }
+
+realPersistenceFor :: (FromJSON msg, ToJSON msg) => String -> FilePath -> IO (MessagePersistence IO msg)
+realPersistenceFor actor tmpDir = do
+  Persistence{load, save} <- createPersistence $ tmpDir </> actor </> "acks"
+  PersistenceIncremental{loadAll, append} <- createPersistenceIncremental $ tmpDir </> actor </> "network-messages"
+
+  pure $
+    MessagePersistence
+      { loadAcks = do
+          mloaded <- load
+          case mloaded of
+            Nothing -> pure $ replicate (length [alice, bob]) 0
+            Just acks -> pure acks
+      , saveAcks = save
+      , loadMessages = loadAll
+      , appendMessage = append
+      }
```

Comment:
This seems to be unused. I'll remove it.

Reply:
thank you, I was multitasking too muc